RoboFile.php

```diff
@@ -302,6 +303,34 @@ public function testCloneSite(string $site_name) {
     return ResultData::EXITCODE_OK;
   }
 
+  /**
+   * Add allow plugins section to composer.
+   *
+   * @param string $site_name
+   *   The machine name of the site to add the allow plugins section to.
+   */
+  public function testAllowPlugins(string $site_name) {
+    $site_folder = $this->getSiteFolder($site_name);
+    chdir($site_folder);
+    $plugins = [
```

Comment:
Not sure if this should match, but the list of plugins currently allowed by the platform (IC) is as follows:
* composer/installers
* cweagans/composer-patches
* drupal/core-composer-scaffold
* dealerdirect/phpcodesniffer-composer-installer
* [not applicable to Drupal] johnpbloch/wordpress-core-installer

Reply:
I think it only matters that this function converts the test fixture to a functioning site, so only plugins used in the fixtures need be listed.